src/dso_api/dynamic_api/routers.py

```diff
@@ -332,6 +338,15 @@ def _build_remote_viewsets(self, api_datasets: Iterable[Dataset]):
 
         return tmp_router.registry
 
+    def _build_doc_views(self, datasets: Iterable[Dataset]) -> Iterator[URLPattern]:
+        for dataset in datasets:
```

Comment:
Moeten we nog rekening houden met `status == 'niet_beschikbaar'` bij generatie van de docs?

Reply:
En we hebben ook nog de `enable_api` vlag.

Reply:
Gebeurt al waar deze methode wordt aangeroepen:

        # Create viewsets only for datasets that have an API enabled
        db_datasets = [ds for ds in db_datasets if ds.enable_api]
        dataset_routes = self._build_db_viewsets(db_datasets)
        # Same for remote API's
        api_datasets = list(get_active_datasets().endpoint_enabled())
        remote_routes = self._build_remote_viewsets(api_datasets)

        datasets = db_datasets + api_datasets

        doc_urls = self._build_doc_views(datasets)